querybook/webapp/components/DataDocSchedule/DataDocScheduleForm.tsx

```diff
@@ -226,6 +286,23 @@ export const DataDocScheduleForm: React.FunctionComponent<IDataDocScheduleFormPr
                         <FormWrapper minLabelWidth="180px" size={7}>
                             <Form>
                                 <DisabledSection disabled={!isEditable}>
+                                    <ExecutionRangeField
+                                        values={formatExecutionRangeValue(
```

Comment:
seems like formatExecutionRangeValue achieves nothing since the `ExecutionRangeField` expects all 3 fields (whether null or not) anyway